packages/components/app/styles/components/table.scss

```diff
@@ -0,0 +1,177 @@
+//
+// TABLE
+//
+//
+$hds-table-border-radius: 6px;
+$hds-table-border-width: 1px;
+$hds-table-inner-border-radius: $hds-table-border-radius - $hds-table-border-width;
+$hds-table-border-color: var(--token-color-border-primary);
+$hds-table-cell-padding-medium: 12px 16px;
+$hds-table-cell-padding-short: 4px 16px;
+$hds-table-cell-padding-tall: 20px 16px;
+
+.hds-table,
+.hds-table-sortable {
+  width: 100%;
+  font-size: var(--token-typography-body-200-font-size);
+  font-family: var(--token-typography-font-stack-text);
+  line-height: var(--token-typography-body-200-line-height);
+  table-layout: fixed;
+  border: $hds-table-border-width solid $hds-table-border-color;
+  border-radius: $hds-table-border-radius;
+  border-spacing: 0;
+}
+
+// TABLE HEADER
+.hds-table__thead {
+  .hds-table__tr {
+    color: var(--token-color-foreground-strong);
+    background-color: var(--token-color-surface-strong);
+
+    .hds-table__th,
+    .hds-table__th-sort {
+      height: 48px;
+      color: var(--token-color-foreground-strong);
+      font-weight: var(--token-typography-font-weight-semibold);
+      font-size: 14px;
+      line-height: 1.4;
+      text-align: left;
+      background-color: var(--token-color-surface-strong);
+      border-top: none;
+      border-right: $hds-table-border-width solid transparent;
+      border-bottom: $hds-table-border-width solid $hds-table-border-color;
+      border-left: none;
+    }
+
+    .hds-table__th {
+      padding: $hds-table-cell-padding-medium;
+    }
+
+    .hds-table__th-sort {
+      padding: 0;
+
+      button {
+        width: 100%;
+        height: 48px;
+        margin: 0;
+        padding: $hds-table-cell-padding-medium;
+        font: inherit;
+        text-align: inherit;
+        background-color: transparent;
+        border: 1px solid transparent;
+        border-radius: 0; // this has to be separately declared on the specific cells that need to be rounded
+
+        &:visited {
+          color: var(--token-color-foreground-strong);
+        }
+
+        &:hover {
+          color: var(--token-color-foreground-strong);
+          background-color: var(--token-color-palette-neutral-200);
+          cursor: pointer;
+        }
+
+        &:focus,
+        &:focus-visible { // using the mixin doesn't work for this one, so we copy the styles
+          border: 1px solid var(--token-color-focus-action-external);
+          outline: 0;
+          box-shadow: inset 0 0 0 1px var(--token-color-focus-action-internal), 0 0 0 3px var(--token-color-focus-action-external);
+        }
+
+        &:active {
+          color: var(--token-color-foreground-strong);
+          background-color: var(--token-color-palette-neutral-300);
+        }
+
+        svg { // in case they don't use flight icons (yet)
+          margin-left: 8px;
+          color: var(--token-color-foreground-action);
+          vertical-align: middle; // we can't use display flex on the button, so we have to align the icon itself instead.
```

Comment:
this should be fixed with the update I'm pushing to support multi-line text 👍

Reply:
@MelSumner notice: with the new layout/elements organization, you have to set the height of the `button` to `100%` or this will happen:
<img width="1062" alt="screenshot_2046" src="https://user-images.githubusercontent.com/686239/201648730-2bb46549-6460-4646-ab53-97c6ce03c45b.png">